python/requirements/ml/requirements_rllib.txt

```diff
@@ -39,3 +39,5 @@ onnxruntime==1.12.0
 tf2onnx==1.12.1
 typer==0.6.1
 rich==12.0.1
+# For using Ray Datasets in Offline API.
+pyarrow==6.0.1
```

Comment:
Should we do the "try_import_" thing here as well rather than addind this to our requirements?

Reply:
I think this is a good idea. `pyarrow` will be in the requirements for `ray[data]`. Writing a `try_import_arrow` is a good way. I will write something over the weekend.

Reply:
@ArturNiederfahrenhorst I added now the `try_import_arrow()`. It might be that we need to also check for a certain version of `pyarrow`, afaicr I needed to downgrade `pyarrow` the last time to make it run.